### PR TITLE
Fix ubuntu standalone ci

### PR DESCRIPTION
**Pull request by agutenkunst**
## Description

Fixes issues with the standalone build. Could be related to some cmake update I guess...

See for example:
https://github.com/PilzDE/psen_scan_v2/runs/2762716878?check_suite_focus=true

### Things to add, update or check by the maintainers before this PR can be merged.
<skip>

### CODE REVIEW

File: .github/workflows/ci_action_standalone.yml [hunk 1]
```diff
@@ -25,12 +25,12 @@ jobs:
...
       shell: bash
       working-directory: ${{runner.workspace}}/build
-      run: cmake $GITHUB_WORKSPACE/standalone -DCMAKE_BUILD_TYPE=$BUILD_TYPE
+      run: cmake $GITHUB_WORKSPACE/standalone -DCMAKE_BUILD_TYPE=$BUILD_TYPE -DBUILD_TESTING=ON
 
     - name: Build
       working-directory: ${{runner.workspace}}/build
       shell: bash
-      run: cmake --build . --config $BUILD_TYPE -DBUILD_TESTING=ON
+      run: cmake --build . --config $BUILD_TYPE
```
Comment (martiniil) [inline .github/workflows/ci_action_standalone.yml:33]:
```suggestion
      run: cmake --build .
```
see https://stackoverflow.com/questions/24460486/cmake-build-type-is-not-being-used-in-cmakelists-txt/24470998#24470998 or https://cmake.org/cmake/help/latest/manual/cmake.1.html#build-a-project

**Review by martiniil — CHANGES_REQUESTED** [review]

**Comment by martiniil** [comment]
Should we also recommend a *Release*-build in the README?

**Comment by martiniil** [comment]
It would have been nice to include a link to a failing ci job, however I could not find any.

**Comment by agutenkunst** [comment]
Link to failing job added into description.

**Review by martiniil — APPROVED** [review]

**Pull request merged**